### PR TITLE
Jcs/check if rate used

### PR DESCRIPTION
**Pull request by jSylvestre**

### CODE REVIEW

File: Harvest.Web/Controllers/RateController.cs [hunk 1]
```diff
@@ -157,33 +148,59 @@ public async Task<ActionResult> Edit(int id, RateEditModel model)
 
 
             var rateToEdit = await _dbContext.Rates.SingleAsync(a => a.Id == id);
+            var rateUsed = await _dbContext.Expenses.AnyAsync(a => a.RateId == rateToEdit.Id);
```
Comment (jSylvestre) [inline Harvest.Web/Controllers/RateController.cs:151]: I pulled in main, so the rateUsed was the only actual change

**Review by srkirkland — CHANGES_REQUESTED** [review]
Why bother?  I don't see that it's worth it

**Comment by jSylvestre** [comment]
@srkirkland It prevents cluttering up the table if they edit it and it hasn't been used yet.
But up to you.

**Comment by srkirkland** [comment]
Extra rows in the database wouldn't bother me one bit

**Comment by jSylvestre** [comment]
Not needed

**Pull request closed without merging**